workspaces/ui/src/components/diff/v2/CaptureManagerPage.js

```diff
@@ -155,52 +154,66 @@ class _AllCapturesStore extends React.Component {
   }
 }
 
-export const AllCapturesStore = withNavigationContext(withRouter(withSpecServiceContext(_AllCapturesStore)));
+export const AllCapturesStore = withRouter(withSpecServiceContext(_AllCapturesStore));
 
-export const CaptureManagerPage = withNavigationContext(({match, baseUrl, specService}) => {
-  return (
-    <AllCapturesStore>
-      <IgnoreDiffStore>
-          <Switch>
-            <Route exact path={routerPaths.diffPage(baseUrl)}
-                   component={() => <CaptureManager specService={specService}/>}/>
-            <Route exact path={routerPaths.diffPageWithCapture(baseUrl)}
-                   component={(props) => <CaptureManager specService={specService}
-                                                         captureId={props.match.params.captureId}/>}/>
+export const CaptureManagerPage = ({match, specService}) => {
+
+  const routerPaths = useRouterPaths();
 
-            <Route exact path={routerPaths.diffRequestNew(baseUrl)} component={(props) => {
-              return (
-                <TrafficSessionStore sessionId={props.match.params.captureId} specService={specService}>
-                  <DiffPageNew {...props} />
-                </TrafficSessionStore>
-              );
-            }}/>
-          </Switch>
-      </IgnoreDiffStore>
-    </AllCapturesStore>
+  return (
+    <Page title="Optic Live Contract Testing Dashboard">
+      <Page.Navbar
+        mini={true}
+        baseUrl={match.url}
+      />
+      <Page.Body>
+        <Switch>
+          <AllCapturesStore>
+            <IgnoreDiffStore>
+              <Switch>
+                <Route exact path={routerPaths.diffPage()}
```

Comment:
let's move these to *Root components. *Root components are glue between react-router and our app components